pages/1.13/release-notes/1.13.0/index.md

```diff
@@ -310,6 +310,131 @@ The issues that have been fixed in DC/OS 1.13 are grouped by feature, functional
 # Known issues and limitations
 This section covers any known issues or limitations that don’t necessarily affect all customers, but might require changes to your environment to address specific scenarios. The issues are grouped by feature, functional area, or component. Where applicable, issue descriptions include one or more tracking identifiers enclosed in parenthesis for reference.
 
+### Using separate JSON files for job scheduling
+In this release, jobs and job schedules are created in two separate steps. Because of this change, you must structure the job definition in the JSON editor in distinct sections similar to this:
+
+- job: JSON definition that specifies the job identifier and job configuration details.
+- schedule: JSON definition the specifies the schedule details for the job.
+
+This two-step approach to creating JSON files for jobs is different from previous releases in which jobs and schedules could be created in one step. In previous releases, the job could have its schedule embedded in its JSON file. 
```

Comment:
Personally I think it might be clearer to replace instances of "JSON files" with "JSON configurations", since the user will be creating and modifying the JSON in the UI instead of saving it as a file on their machine.

Reply:
done